packages/tools/generator-component/generators/app/templates/__package__.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@justeat/f-<%= name.default %>",
-  "description": "Fozzie <%= name.readme %> – <%= description %>",
+  "description": "Fozzie <%= name.readme %> - <%= description %>",
   "version": "0.1.0",
   "main": "dist/f-<%= name.default %>.umd.min.js",
   "files": [
-    "dist"<% if(needsComponentTests) { %>,
-    "test-utils"<% } %> <% if(needsTestingApiMocks) { %>,
+    "dist"<% if(config.needsComponentTests) { %>,
+    "test-utils"<% } %> <% if(config.needsTestingApiMocks) { %>,
     "src/services"<% } %>
   ],
   "homepage": "https://github.com/justeat/fozzie-components/tree/master/packages/f-<%= name.default %>",
```

Comment:
This field needs to be updated to match the new project structure.

Reply:
Fixed